build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/ElasticsearchJavaPlugin.java

```diff
@@ -111,119 +96,57 @@ public static void configureConfigurations(Project project) {
             configuration.resolutionStrategy(ResolutionStrategy::failOnVersionConflict);
         });
 
-        // force all dependencies added directly to compile/testImplementation to be non-transitive, except for ES itself
```

Comment:
that was never used

Reply:
I guess this is handled by the bit below?